src/core/ColorPicker/ColorPicker.tsx

```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils/hooks/useTheme';
+import '@itwin/itwinui-css/css/color-picker.css';
+import { CommonProps } from '../utils/props';
+import cx from 'classnames';
+
+export const DefaultColors = [
+  { color: '#FFFFFF', name: 'WHITE' },
+  { color: '#5A6973', name: 'GREY' },
+  { color: '#00121D', name: 'KURETAKE BLACK MANGA' },
+  { color: '#002A44', name: 'RHAPSODY IN BLUE' },
+  { color: '#00426B', name: 'DARK IMPERIAL BLUE' },
+  { color: '#005A92', name: 'JETSKI RACE' },
+  { color: '#0073BA', name: 'FRENCH BLUE' },
+  { color: '#008BE1', name: 'BLUE COLA' },
+  { color: '#30B0FF', name: 'FANTASY CONSOLE SKY' },
+  { color: '#58BFFF', name: 'HELLO SUMMER' },
+  { color: '#7FCEFF', name: 'CHROMIS DAMSEL BLUE' },
+  { color: '#A6DDFF', name: 'DROPLET' },
+  { color: '#CDECFF', name: 'LUCID DREAMS' },
+  { color: '#E5F5FD', name: 'KODAMA WHITE' },
+  { color: '#010200', name: 'REGISTRATION BLACK' },
+  { color: '#122306', name: 'YUZU SOY' },
+  { color: '#23450b', name: 'FOREST GREEN' },
+  { color: '#346711', name: 'TATZELWURM GREEN' },
+  { color: '#458816', name: 'CHLOROPHYLL' },
+  { color: '#56AA1C', name: 'PLASTIC PINES' },
+  { color: '#5FBB1F', name: 'FIELD GREEN' },
+  { color: '#67CC22', name: 'GREEN HIGH' },
+  { color: '#91E458', name: 'LILLIPUTIAN LIME' },
+  { color: '#B2EC8B', name: 'GREEN DAY' },
+  { color: '#D4F4BD', name: 'TEA GREEN' },
+  { color: '#EEF6E8', name: 'VERDE PASTEL' },
+  { color: '#9BA5AF', name: 'SERYI GREY' },
+  { color: '#CF0000', name: 'RED EPIPHYLLUM' },
+  { color: '#FF6300', name: 'SAFETY ORANGE' },
+  { color: '#FFC335', name: 'RISE-N-SHINE' },
+];
+
+export type ColorPickerProps = {
+  /**
+   * Color Swatches that will be added within the color palette.
+   */
+  children?: React.ReactNode;
+} & Omit<CommonProps, 'title'>;
+
+export const ColorPicker = (props: ColorPickerProps) => {
+  const { children, className, ...rest } = props;
+
+  useTheme();
+
+  return (
+    <div className={cx('iui-color-picker', className)} {...rest}>
+      <div className='iui-color-palette'>{children}</div>
```

Comment:
I was expecting color palette logic to be part of this component.
- changing active color on click/enter/space
- `onChange` prop to know when active color changes
- arrow key navigation

I see you have some basic click handling in the story. Why not handle everything here?

Reply:
I had originally tried to handle all of that within the component but decided to remove it.
After looking at some other existing examples of color picker in react it looked like they did not have the color picker automatically setting the active color, they only had the onClick handler when clicking on a color and leaving the setting active up to the user. 
Also setting active automatically got tricky trying to handle situations when they passed in anything other than a ColorSwatch. For example in the WithTooltip story the ColorSwatches are wrapped in Tooltips so the automatic active color will not work because the ColorPicker was looking for ColorSwatches in the children but found Tooltips. There will be the same problem if any other custom component is passed in.

Reply:
> After looking at some other existing examples of color picker in react it looked like they did not have the color picker automatically setting the active color

Well, you will need to handle the color selection for advanced picker (by providing an `onChange` prop). And the saved color palette in advanced picker would most certainly need to behave consistently with the slider. So you may want to do some more research and experimentation (just so that we don't need to reverse any decisions we make now).

> setting active automatically got tricky trying to handle situations when they passed in anything other than a ColorSwatch.

From what I can tell, custom components are really only needed for tooltips, so maybe we could try making tooltips part of the API and see if it offers any benefits.

> they only had the onClick handler when clicking on a color and leaving the setting active up to the user.

If you want to keep this approach, you should still be able to improve the keyboard aspect of it, by calling `onClick` whenever enter/spacebar is pressed.